Sources/Apodini/Request/DynamicProperty.swift

```diff
@@ -0,0 +1,214 @@
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "DynamicProperty \(property.name) on element \(info.name) must be a struct")
+
+                    dynamicProperty.execute(operation)
+                case let dynamics as Traversible:
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "Dynamics \(property.name) on element \(info.name) must be a struct")
+                
+                    dynamics.execute(operation)
+                default:
+                    break
+                }
+            }
+        } catch {
+            fatalError("Executing operation on all properties of \((try? typeInfo(of: Target.self))?.name ?? "Unknown Type") on DynamicProperty \((try? typeInfo(of: Self.self))?.name ?? "Unknown Type") failed.")
+        }
+    }
+    
+    mutating func apply<Target>(_ mutation: (inout Target, _ name: String) -> Void) {
+        do {
+            let info = try typeInfo(of: Self.self)
+
+            for property in info.properties {
+                let child = try property.get(from: self)
+
+                switch child {
+                case var target as Target:
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "\(Target.self) \(property.name) on element \(info.name) must be a struct")
+                    
+                    mutation(&target, property.name)
+                    try property.set(value: target, on: &self)
+                case var dynamicProperty as DynamicProperty:
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "DynamicProperty \(property.name) on element \(info.name) must be a struct")
+                    
+                    dynamicProperty.apply(mutation)
+                    try property.set(value: dynamicProperty, on: &self)
+                case var dynamics as Traversible:
+                    assert(((try? typeInfo(of: property.type).kind) ?? .none) == .struct, "Dynamics \(property.name) on element \(info.name) must be a struct")
+                
+                    dynamics.apply(mutation)
+                    try property.set(value: dynamics, on: &self)
+                default:
+                    break
+                }
+            }
+        } catch {
+            fatalError("Applying operation on all properties of \((try? typeInfo(of: Target.self))?.name ?? "Unknown Type") on DynamicProperty \((try? typeInfo(of: Self.self))?.name ?? "Unknown Type") failed.")
+        }
+    }
+}
+
+
+// MARK: Dynamics Implemenation
+
+private protocol Traversible {
+    func execute<Target>(_ operation: (Target, _ name: String) -> Void)
+    
+    mutating func apply<Target>(_ mutation: (inout Target, _ name: String) -> Void)
+}
+
+extension Dynamics: Traversible {
+    func execute<Target>(_ operation: (Target, _ name: String) -> Void) {
+        for (name, element) in self.elements {
```

Comment:
As `Dynamics` conforms to collection you can remove `.element`
```suggestion
        for (name, element) in self {
```

Reply:
True, that was an artifact from the initial version (where `Dynamics` was no `Collection`) I forgot to remove.